webapp/components/wpt-results.html

```diff
@@ -628,6 +628,11 @@ <h3>
                     total: 0,
                   })),
                 };
+                row.diff = {
```

Comment:
These and some following changes seem unrelated to this PR. Can you split out?

Reply:
Bad merge. reverted.